src/navigations/HomeBottomTabs.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-use-before-define */
 import * as React from 'react';
+import PushNotification from 'react-native-push-notification';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `PushNotification` from 'react-native-push-notification' is added but it seems to be unused in the rest of the code. Please ensure that you remove unused imports to keep the code clean and maintainable.
```diff
- import PushNotification from 'react-native-push-notification';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->